gcloud-java-compute/src/test/java/com/google/gcloud/compute/ComputeImplTest.java

```diff
@@ -1425,6 +1457,445 @@ public void testDeleteZoneOperation_False() {
   }
 
   @Test
+  public void testGetGlobalAddress() {
```

Comment:
Do we test somewhere that we get null if the address doesn't exist? (also applies to other `get` methods)